### DISCUSSION TITLE
how to use kyuubi with hue, there is no kyuubi attribute in hue.ini file

### DISCUSSION THREAD

**Discussion by chengduhaha** [cloudera/hue#3565] [Questions] [answered]
I want to use kyuubi with hue, don't know how to write config file hue.ini. my hue version is 4.11.0

**Comment by Harshg999** [comment, marked as answer]
Thanks @chengduhaha for contributing Hue-Kyuubi support with JDBC interface via https://github.com/cloudera/hue/pull/3586.
It's currently merged in master branch and will also be available in upcoming Hue release.
